src/constants/WalletAndCurrencyConstants.js

```diff
@@ -547,12 +547,20 @@ export const SPECIAL_CURRENCY_INFO: {
   eboost: {
     initWalletName: s.strings.string_first_eboost_wallet_name,
     chainCode: 'EBST',
+    isImportKeySupported: {
```

Comment:
No need to add to eboost since it's in 'keysOnlyMode' mode so it won't appear anyway.

Reply:
Keeping it just incase we reenable eboost